pkg/controller/idler/idler_controller.go

```diff
@@ -126,11 +107,20 @@ func (r *ReconcileIdler) Reconcile(request reconcile.Request) (reconcile.Result,
 			"failed to ensure idling '%s'", idler.Name)
 	}
 	// Find the earlier pod to kill and requeue. Do not requeue if no pods tracked
-	requeue, after := nextPodToBeKilledAfter(idler)
-	log.Info("requeueing in", "duration", after)
+	nextTime := nextPodToBeKilledAfter(idler)
+	if nextTime == nil {
```

Comment:
```suggestion
	if nextTime == nil || *nextTime <0 {
```

Reply:
I added this suggestion before you added the check to `nextPodToBeKilledAfter()` ;)